src/testingtool/TestingTool.java

```diff
@@ -14,14 +14,14 @@
     private TanTest tanTest;
     private TrigTest trigTest;
 
-    private final double e = 0.000001;  //  e = 1x10^-6
+    private final double e = 0.0000001;  //  e = 1x10^-6
```

Comment:
this is 1x10^-7, comment should be updated to reflect this

Reply:
Ok